api/src/main/java/org/pmiops/workbench/api/CohortsController.java

```diff
@@ -136,6 +139,16 @@ private void checkForDuplicateCohortNameException(String newCohortName, DbWorksp
         workspaceNamespace, workspaceId, WorkspaceAccessLevel.WRITER);
     DbWorkspace workspace = workspaceService.getRequired(workspaceNamespace, workspaceId);
 
+    try {
+      // validate the cohort definition
+      new Gson().fromJson(cohort.getCriteria(), SearchRequest.class);
```

Comment:
Is there any chance that cohort could be null? Should we also handle nullPointer?

Reply:
UI doesn't allow for saving a null cohort and i think swagger would throw an exception if the request body was null.